test/NuGet.Core.FuncTests/Dotnet.Integration.Test/DotnetSourcesTests.cs

```diff
@@ -236,40 +259,38 @@ public void Sources_WarnWhenListHttpSource(string initialSource, string secondSo
                 CommandRunnerResult result = _fixture.RunDotnetExpectSuccess(configFileDirectory, string.Join(" ", args));
 
                 // Assert
-                Assert.Equal(shouldWarn, result.Output.Contains(warningMessage));
+                Assert.Contains(initialSource, result.AllOutput);
```

Comment:
The nuget list command, allows users to list configured sources. However, just because a user has an http source, we should not be throwing an error for the list command. Instead, we should list all the configured sources and warn them about it. As a result, I have not added an error for the list command.